utils/index.js

```diff
@@ -61,7 +61,10 @@ const checkPublic = function(req, res, next) {
   // check if user exists in session (ie, by checking for firstName)
   const { personal: { firstName = null } = {} } = req.session
   if (!firstName) {
-    req.session = API.getUser('A5G98S4K1')
+    const user = API.getUser('A5G98S4K1')
+
+    // setting req.session = {obj} causes an error, so assign the keys one at a time
+    Object.keys(user).map(key => (req.session[key] = user[key]))
```

Comment:
I can't believe I didn't think of this when I was working on memorystore 🤦‍♀

Reply:
Yeah, I was briefly considering changing all data to a new key (`session.user`, maybe?).

Anyway, that would have been awful.